third-party/dms-guild.index

```diff
@@ -15,6 +15,10 @@
 		<!-- feats -->
 		<file name="expanded-racial-feats.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/dms-guild/expanded-racial-feats.xml" />		
 		
+		<!-- classes -->
+		<file name="class-cook.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/dms-guild/class-cook.xml" />
```

Comment:
when updating existing files, please update the version number at the top of the file